lib/console_yasttest.pm

```diff
@@ -5,20 +5,29 @@ use base 'y2logsstep';
 use strict;
 use testapi;
 
-sub assert_service_widget {
+sub assert_service_configuration {
     my ($self, %args) = @_;
-    my $after_writing_conf = $args{after_writing_conf} || 'alt-f';
-    my $after_reboot       = $args{after_reboot}       || 'alt-a';
+    my $after_writing_conf_ref = $args{after_writing_conf} || [qw(alt-t keep_current_state)];
```

Comment:
I guess it's a bit of overkill to use an array here, with 2 variables it would be more clear and readable. Here we rely on the order, don't check if array element exists at all. I also see that we always pass the same pairs, meaning that we can even map keys in the method itself depending on the action.

Reply:
The use of the array is to make it readable in the call (I tried with a hash and it was more cumbersome as it would be a hash inside a hash). I am not sure I got your point about the 2 variables, we need some structure to store the shortcut and the field, in that way we don't need to declare extra variables in the call.
To check the element allows to save some typing for the default values of the widget. 
The pairs are different, that I could find, for example for after_writing_conf: alt-t, alt-f and alt-e. I like the idea to map keys inside the method and not repeat keys but they don't depend on the action, they depends on the caller, so it would be an extra parameter and some switch inside which I don't see it more clean.